File: src/main/java/gr/apt/repository/notification/NotificationViewersRepository.java
package gr.apt.repository.notification;

import gr.apt.persistence.entity.notification.NotificationViewers;
import io.quarkus.hibernate.orm.panache.PanacheRepositoryBase;

import javax.enterprise.context.ApplicationScoped;
import java.math.BigInteger;
@ApplicationScoped
public class NotificationViewersRepository implements PanacheRepositoryBase<NotificationViewers, BigInteger> {
}

File: src/main/java/gr/apt/repository/notification/NotificationViewersRepository.kt
package gr.apt.repository.notification

import gr.apt.persistence.entity.notification.NotificationViewers
import io.quarkus.hibernate.orm.panache.PanacheRepositoryBase
import java.math.BigInteger
import javax.enterprise.context.ApplicationScoped

@ApplicationScoped
class NotificationViewersRepository : PanacheRepositoryBase<NotificationViewers?, BigInteger?>